020_IoCWithUnity/Bootstrapper.cs

```diff
@@ -15,7 +15,7 @@ public static class Bootstrapper
         public static void SetupContainer(IUnityContainer unityContainer)
         {
             // TODO: Register your dependencies here
-            // unityContainer.RegisterType<IInterface, Implementor>(...)
+            // unityContainer.RegisterType<IInterface, Implementor>();

```

Comment:
Please revert this change. The original purpose was to demonstrate overloads of the method